packages/management/featuregates/bundle/config/upstream/rbac.yaml

```diff
@@ -3,12 +3,16 @@ kind: ServiceAccount
 metadata:
   labels:
     app: tanzu-featuregates-manager
+  annotations:
+    kapp.k14s.io/change-group: "featuregates.config.tanzu.vmware.com/serviceaccount"
```

Comment:
dont think we need to add SA to a group.

Reply:
ClusterRoleBinding has a reference to this SA, so just to be safe I added this to the group as well even though I did not see an error with SA and also i think it doesn't harm anything, but if you are sure that removing this annotation on SA wouldn't cause the issue I can remove it(the problem is its hard to test this fix as i don't hit this issue always)

Reply:
let's leave this as-is for now and can revisit if needed later. Also, I think kapp-controller will be fixing this in its new version, so we can come revisit then if needed